charts/nfs-subdir-external-provisioner/templates/deployment.yaml

```diff
@@ -24,6 +24,8 @@ spec:
         {{- include "nfs-subdir-external-provisioner.podLabels" . | nindent 8 }}
     spec:
       serviceAccountName: {{ template "nfs-subdir-external-provisioner.serviceAccountName" . }}
+      securityContext:
+      {{- toYaml .Values.podSecurityContext | nindent 8 }}
```

Comment:
one more tab is required (indentation)

Reply:
I am not sure how to fix the author issue, so I am going to just re-create this merge request, sorry!